src/components/MaximWPproj2/CentCont.styled.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { ButtonStyle, CentreCont, Detailsincont, Hrstyle2, StyledHeader, StyledNumber } from './styles/Container.styled'
+
+type Props = {
+  title: string
+  number: string
+  text1: string
+  text2: string
+  text3: string
+}
+
+export const CentCont = (props: Props) => {
+  return (
+    <CentreCont>
+      <StyledHeader containertype={'Main'}>{props.title}</StyledHeader>
+      <StyledNumber containertype={'Main'}>{props.number}</StyledNumber>
+      <Detailsincont containertype={'Main'}>
+        <Hrstyle2></Hrstyle2>
```

Comment:
can use <Hrstyle2 /> instead since there are no children and should be StyledHr2? or at least HrStyle2 (PascalCase)